packages/bap-cwe-checker/bap-cwe-checker.master/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+name: "bap-cwe-checker"
+version: "master"
+maintainer: "Thomas Barabosch"
+authors: "Thomas Barabosch"
+homepage: "https://github.com/fkie-cad/cwe_checker"
+bug-reports: "https://github.com/fkie-cad/cwe_checker"
+license: "LGPL-3.0"
+install: [
+  ["make"]
+  ["mkdir" "-p" "%{share}%/bap-cwe-checker/"]
+  ["cp" "src/config.json" "%{share}%/bap-cwe-checker/"]
+]
+
+remove: [
+  ["make" "uninstall"]
+  ["rm" "-r" "%{share}%/bap-cwe-checker/"]
+]
+
+depends: [
+  "bap"
```

Comment:
the readme specifies concrete version of the packages, please use those versions